testing/file_test/file_test_base.cpp

```diff
@@ -720,18 +725,32 @@ static auto Main(int argc, char** argv) -> int {
     return EXIT_FAILURE;
   }
 
+  // Tests might try to read from stdin. Ensure those reads fail by closing
+  // stdin and reopening it as /dev/null.
+  llvm::sys::Process::SafelyCloseFileDescriptor(0);
```

Comment:
```suggestion
  llvm::sys::Process::SafelyCloseFileDescriptor(STDIN_FILENO);
```

Reply:
That's a POSIX extension that doesn't exist on Windows.

Reply:
Well, fixup is anyway. Maybe it's okay.